### PR TITLE
feat: Add properties story

### LINKED ISSUE #718: Display properties for metadata, back matter, and controls
### Description

As a governance, risk, and compliance author, I would like to see a generic display of properties for metadata, backmatter and controls.

### Acceptance Criteria

* The following `props` fields are properly displayed within the React library for Metadata (`metadata/props`), Backmatter (`backmatter/resources/props`), and Controls (`controls/props`):
   * `name` [1]
   * `ns` [0 or 1]
   * `value` [1]
   * `class` [0 or 1]
   * `remarks` [0 or 1]

### Proposed Solution

The data will be displayed in a table in a modal that can be launched by clicking an icon on the field(s).

### Additional Notes

`markings` will be handled separately. For more information see: https://pages.nist.gov/OSCAL/reference/latest/complete/json-reference/#/profile/metadata/props

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

### PR DESCRIPTION
**Pull request by Bronstrom**
This adds a properties story in storybook.

Closes #718

### CODE REVIEW

File: packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  {
+    name: "Example Property 2",
+    value: "Example Value",
+  },
+  {
+    name: "Example Property 3",
+    value: "Example Value",
+  },
+];
```
Comment (Bronstrom) [inline packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx:20]: For our storybook content, would we rather include our data within where we're defining the story or keep all data to a seperate file?

Comment (kylelaker) [reply packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx:20]: i like this better

**Comment by tuckerzp** [comment]
> However, looks like this poses an issue to the linter:

Not sure if this is correct, but what happens if you give the linter what it wants?

File: packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx (hunk 1 above)
Comment (tuckerzp) [reply packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx:20]: I think for something so small and specific to the `Properties` storybook page, it is ok to put it in the file itself.

**Comment by kylelaker** [comment]
> Not sure if this is correct, but what happens if you give the linter what it wants?

To explain why this is bad idea, Storybook is not a run time dependency so should be listed in `devDependencies`. The linter doesn't realize Stories are special files that aren't in the production build.

**Review by kylelaker — APPROVED** [review]

**Review by kylelaker — APPROVED** [review]

**Review by tuckerzp — APPROVED** [review]

**Pull request merged**